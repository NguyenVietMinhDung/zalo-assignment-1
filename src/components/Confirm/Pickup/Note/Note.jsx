import React from 'react';
import { Row, Col } from 'react-grid-system';
import './Note.scss';

const Note = () => (
  <Row>
    <Col className="note" xs={12}>
      <div className="note__icon">
        <i className="lni lni-16 lni-notepad" />
      </div>
      <div className="note__input">
        <input type="text" placeholder="Nhập ghi chú" />
      </div>
    </Col>
  </Row>
);

export default Note;
