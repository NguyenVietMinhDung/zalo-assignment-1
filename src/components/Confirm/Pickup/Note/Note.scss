@import '../../../../styles/variables';

.note {
	display: flex;

	padding-top: 15px;
	padding-bottom: 15px;

	&__icon {
		margin-right: 5px;

		i {
			opacity: 0.2;

			font-weight: bold;
		}
	}

	&__input {
		flex: 1;

		input {
			width: 100%;

			border: 0;

			&:focus {
				outline: none;
			}

			&::placeholder {
				color: lightgray;

				font-weight: bold;
			}
		}
	}
}
