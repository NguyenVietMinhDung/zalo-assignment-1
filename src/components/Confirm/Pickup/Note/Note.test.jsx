import React from 'react';
import { shallow } from 'enzyme';
import Note from './Note';

describe('<Note />', () => {
  test('renders correctly', () => {
    const component = shallow(<Note />);
    expect(component).toMatchSnapshot();
  });
});
