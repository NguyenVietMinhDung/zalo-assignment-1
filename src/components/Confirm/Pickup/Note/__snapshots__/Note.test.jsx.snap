// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Note /> renders correctly 1`] = `
<Row
  align="normal"
  component="div"
  debug={false}
  gutterWidth={null}
  justify="start"
  nogutter={false}
  nowrap={false}
  style={Object {}}
>
  <Col
    className="note"
    component="div"
    debug={false}
    lg={null}
    md={null}
    offset={Object {}}
    pull={Object {}}
    push={Object {}}
    sm={null}
    style={Object {}}
    width={null}
    xl={null}
    xs={12}
    xxl={null}
  >
    <div
      className="note__icon"
    >
      <i
        className="lni lni-16 lni-notepad"
      />
    </div>
    <div
      className="note__input"
    >
      <input
        placeholder="Nhập ghi chú"
        type="text"
      />
    </div>
  </Col>
</Row>
`;
